corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -755,18 +763,42 @@ hqDefine('geospatial/js/models', [
             self.loadSelectedPolygonFromQueryParam();
         };
 
-        self.exportGeoJson = function (exportButtonId) {
-            const exportButton = $(`#${exportButtonId}`);
-            const selectedId = parseInt(self.selectedSavedPolygonId());
-            const selectedPolygon = self.savedPolygons().find(
-                function (o) { return o.id === selectedId; }
-            );
-            if (selectedPolygon) {
-                const convertedData = 'text/json;charset=utf-8,' + encodeURIComponent(JSON.stringify(selectedPolygon.geoJson));
-                exportButton.attr('href', 'data:' + convertedData);
-                exportButton.attr('download','data.geojson');
+        self.saveGeoPolygon = function () {
+            let data = self.mapObj.drawControls.getAll();
+            if (data.features.length) {
+                let name = window.prompt(gettext("Name of the Area"));
+                if (!name) {
+                    return;
```

Comment:
Does this close the input popup entirely? If so, it might be good to let the user know what happened otherwise they might think the polygon got saved successfully.

Reply:
Addressed in 633b6e760fa7e08c67d29e4b48e0f4aa19892724